### ISSUE TITLE
Error: connection not open | System Error when running Ganache 2.0.0 on win32

### ISSUE THREAD

**Issue by sidkasat** [trufflesuite/ganache#1238]
PLATFORM: win32
GANACHE VERSION: 2.0.0

EXCEPTION:
```
Error: connection not open
    at WebsocketProvider.send (C:\Users\SIKASAT\AppData\Local\Programs\Ganache\resources\app.asar\node_modules\web3-providers-ws\src\index.js:276:18)
    at Timeout._onTimeout (C:\Users\SIKASAT\AppData\Local\Programs\Ganache\resources\app.asar\node_modules\web3-providers-ws\src\index.js:261:19)
    at ontimeout (timers.js:469:11)
    at tryOnTimeout (timers.js:304:5)
    at Timer.listOnTimeout (timers.js:264:5)
    at listOnTimeoutNT (timers.js:340:26)
    at _combinedTickCallback (internal/process/next_tick.js:135:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

APPLICATION LOG:
```
T+0ms: Starting server (version 2.0.0) with initial configuration: {"gasLimit":6721975,"gasPrice":20000000000,"hardfork":"petersburg","hostname":"0.0.0.0","port":7545,"network_id":5777,"default_balance_ether":100,"total_accounts":10,"unlocked_accounts":[],"locked":false,"vmErrorsOnRPCResponse":true,"verbose":false,"db_path":"C:\\Users\\SIKASAT\\AppData\\Roaming\\Ganache\\workspaces\\yellow-bucket\\chaindata"}
T+246ms: Ganache started successfully!
T+246ms: Waiting for requests...
```

**Comment by davidmurdoch** [comment]
This should now be fixed in 2.0.1-beta.0! Do you mind trying to beta and letting me know if that works for you.

https://github.com/trufflesuite/ganache/releases/tag/v2.0.1-beta.0

**Comment by davidmurdoch** [comment]
Fixed in https://github.com/trufflesuite/ganache/releases/tag/v2.3.0! Please let me know if you are still experiencing problems!
